t/exit.t

```diff
@@ -10,7 +10,7 @@ log_level('warn');
 repeat_each(120);
 #repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * (blocks() * 5) - (repeat_each() * 2);
```

Comment:
Doing so would be less resistant to changes made to `repeat_each()`, and would require manual updates to the plan each time the repeat value is changed. Is that desired?